tools/Deployment/gulpfile.js

```diff
@@ -243,9 +248,10 @@ gulp.task("syncBranchCore", () => {
     let docfxHome = path.resolve(config.docfx.home);
     return SyncBranch.runAsync(repoUrl, docfxHome, config.sync.fromBranch, config.sync.targetBranch);
 });
-gulp.task("test", gulp.series("clean", "build", "e2eTest", "publish:myget-test"));
-gulp.task("dev", gulp.series("clean", "build", "e2eTest"));
-gulp.task("dev:release", gulp.series("clean", "build", "e2eTest", "publish:myget-dev", "publish:azdevops-perf-login", "publish:azdevops-perf", "publish:azdevops-internal-login", "publish:azdevops-internal", "publish:azdevops-ppe-login", "publish:azdevops-ppe"));
+gulp.task("test", gulp.series("clean", "build", "pack", "e2eTest", "publish:myget-test"));
```

Comment:
I am not sure whether we still use `test` task. If used, we can put `pack` after `e2eTest`. We only need to pack them when e2eTest runs successfully. Same for `dev:release`.